devito/operator/operator.py

```diff
@@ -131,7 +131,7 @@ class Operator(Callable):
     refer to the relevant documentation.
     """
 
-    _default_headers = ['#define _POSIX_C_SOURCE 200809L']
+    _default_headers = [('_POSIX_C_SOURCE', '200809L')]
```

Comment:
Why is #define dropped?

Reply:
it's added by the actual code generator in the last stage of the compilation pipeline. The information provided here is now more structured than before, as the define _name_ and _value_ are explicitly separated